src/utils/local.ts

```diff
@@ -104,15 +107,46 @@ export class LocalRelease extends EventEmitter implements Release {
     if (path.endsWith(configFile)) {
       const conf = this.getConfig()
       this.config = conf.config
-      this.files = conf.files || []
       this.app = conf.app || conf.app_id || ""
+      this.getFiles(conf.files)
     } else if (path.endsWith(secretsFile)) {
       this.secrets = this.getSecrets()
     }
     this.emit('update', this)
   }
+
+  private getFiles (files: string[] | undefined) {
+    if (!files) return
+
+    const conf = this.getConfig()
+    if (conf.files) conf.files.forEach((file) => {
+      if (!this.isIn(this.files, file)) this.files.push(file)
+    })
+
+    files.forEach((file) => {
+      glob(file, {}, (error, filesFound) => {
+        if (error) throw error
+
+        filesFound = filesFound.map((e) => this.removePrefix(e))
+        filesFound.forEach((file) => {
+          if (!this.isIn(this.files, file)) this.files.push(file)
+        })
+      })
+    })
+  }
+
+	private isIn (array:string[], name:string) {
+		array.forEach((e) => {
+			if (e === name) return true
+		})
+		return false
+	}
```

Comment:
I almost missed this function because it is indented way differently ;)

Reply:
@jeromegn I had not realized until now but, ever since I updated my vim configuration on Friday, all my files spacing has been really messed up and atom did not display the new spacing. Sorry about that - I think I fixed it on all the files that got indented incorrectly.